core/src/main/java/com/github/kaktushose/jda/commands/annotations/interactions/AutoComplete.java

```diff
@@ -44,6 +44,20 @@
 ///         event.replyChoices(...);
 ///     }
 ///     ```
+///
+/// **Be aware that the example above will register *every* command option with auto complete enabled.** If you want to
+/// avoid that, you have to explicitly state the command options the handler supports:
+/// ```java
+/// @SlashCommand("favourite food")
+/// public void foodCommand(CommandEvent event, String fruit, String vegetable) {
+///     event.reply("You've chosen: %s and %s", fruit, vegetable);
+/// }
+///
+/// @AutoComplete(vale = "foodCommand", options = "fruit")
+/// public void onFruitAutoComplete(AutoCompleteEvent event) {
+///     event.replyChoices(...);
+/// }
+/// ```
```

Comment:
We should mention that multiple auto complete methods are possible. And that a "no option" auto complete will be used as the fallback